### ISSUE TITLE
Inconsistent behaviour and presentation of controls

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#565]
Issue group: Buttons
Visibility: 3
Evaluation: Partially Supports
Success Criteria: 3.2.4: Consistent Identification
Needs design: Interaction

### Description
Buttons should perform an action, make a change on the current page or submit a request. Links should be used for navigation. The behaviour across the site is rather inconsistent. Links and buttons are both used for actions and navigation. Links are presented with styling that may appear like a button or like a link regardless of operation (although consistency between visual cues and behaviour is encouraged). 

- [x] Audit where we use buttons and links
- [ ] Code and dev work is not complex (replace links with buttons, vice versa)

![]()

## Related Airtable links
[Link 1](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/rec2cgxLBQYQhncRN?blocks=hide)

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/565?planning-poker) with Zenhub @amazingphilippe @andrewleith @smcmurtry @YedidaZalik

**Comment by amazingphilippe** [comment]
Started the audit in https://www.figma.com/file/alnDnSW1Efi3Lae3uHf6Pj/Component-library-%7C-Biblioth%C3%A8que-de-composantes?node-id=3616%3A1521

**Comment by YedidaZalik** [comment]
Working on audit and trying to find examples, will share with content to add to accessibility stmt when appropriate

**Comment by amazingphilippe** [comment]
Button and link documentation was demoed. Next steps can happen during story refinement to find out if we 
1. Have a design solution
2. Have a tech solution
3. Disclose of the issue in the a11y statement

We can also have a mixed approach

**Comment by amazingphilippe** [comment]
More details from the 2023 audit:

In a number of cases links are communicated as buttons. 

E.g., On the “Browse Templates” page, the “Create Template” and “Copy Template” actions function as links to other pages. However, they are buttons which navigate to those pages when activated. It would be better to have them appear and function as links. See Other Comments section for further suggestions. 

E.g., the mobile navigation menu button has a link role. 
<img width="880" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeHlZQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--92967bcd40628fd876a1f2213d6ade57877fea05/image.png" alt="image.png" />

**Comment by YedidaZalik** [comment]
Including in Ally stmt

**Comment by andrewleith** [comment]
- We could work on a doc to know when to use a button vs. link (can check out design system for ideas)

**Comment by amazingphilippe** [comment]
- [ ] Getting in touch with Pete for advice
